src/exiv2.1

```diff
@@ -278,12 +278,47 @@ not rename the file (overrides \fB\-k\fP). This option is only used
 with the 'rename' action. Note: On Windows you may have to set the TZ
 environment variable for this option to work correctly.
 .TP
-.B \-f
-Do not prompt before overwriting existing files (force overwrite).
+.B \-f,\-F
+These options are used by commands 'rename' and 'extract' to determine file overwrite policy.  These options
+are usually combined with -v/--verbose to provide additional status output.
+.br
+.sp 1
+The options --force and --Force apply to the 'rename' command.  The 'extract' command treats --force and --Force as permission to overwrite.
+.br
+.sp 1
+Default behaviour is to prompt user.
+.br
+-f = Do not prompt before overwriting existing files.
+.br
+-F = Do not prompt before renaming files. Appends '_1'
+('_2', ...) to the name of the new file.  For example:
+.sp 1
+.nf
+$ curl --silent -O http://clanmills.com/Stonehenge.jpg
+$ exiv2 --verbose --Force rename Stonehenge.jpg
+File 1/1: Stonehenge.jpg
+Renaming file to ./20150716_153854.jpg
+$ curl --silent -O http://clanmills.com/Stonehenge.jpg
+$ exiv2 --verbose --Force rename Stonehenge.jpg
+File 1/1: Stonehenge.jpg
+Renaming file to ./20150716_153854_1.jpg
+.sp 1
+.fi
+The 'rename' command will only overwrite files when option --force is used.  The option --Force is provided to avoid unintentional loss of valuable image files.
+.sp 1
```

Comment:
So it doesn't make a difference for 'extract' whether -f or -F is used?

Reply:
For command `extract`, the behaviour of options `--force` and `--Force` are identical.

For command `rename`, the option `--force` grants permission to overwrite.

For command `rename`, the option `--Force` appends _N to the generated filename to ensure that image files are not overwritten.